landoapi/repos.py

```diff
@@ -91,53 +99,65 @@ def __post_init__(self):
             self.short_name = self.tree
 
     @property
-    def phab_identifier(self) -> str:
+    def phab_identifier(self) -> str | None:
```

Comment:
Out of scope for this PR, but I think the way we lay out the repos is not great, since there is a duplication of values in the keys of the dictionary of repos and `short_name`, in each entry. In the future it might be good to refactor this so that there is no duplication.